apps/angular-example/src/app/abby.ts

```diff
@@ -5,6 +5,7 @@ import { environment } from "../environments/environment";
 export const abby = {
   projectId: environment.ABBY_PROJECT_ID,
   currentEnvironment: "test",
+  environments: ["test", "prod"] as Array<string>,
```

Comment:
?
```suggestion
  environments: ["test", "prod"],
```

Reply:
I need to change this file around. The variable is cast as const and therefore the env will be infered to `ReadOnlyArray<string>` which isn't what createAbby takes in as an argument. Good catch tho!